lib/permissions/src/lib.rs

```diff
@@ -26,20 +26,23 @@ type Result<T> = result::Result<T, Error>;
 pub enum ObjectType {
     User,
     Workspace,
+    System,
 }
 
 #[derive(Clone, Copy, strum::Display)]
 #[strum(serialize_all = "snake_case")]
 pub enum Permission {
     Approve,
     Manage,
+    Administer,
 }
 
 #[derive(Clone, Copy, strum::Display, Debug)]
 #[strum(serialize_all = "snake_case")]
 pub enum Relation {
     Approver,
     Owner,
+    Admin,
```

Comment:
You know - I actually wonder if `owner` can also just work here. Only it'd be `owner` of the system vs the workspace